teachopencadd/talktorials/T016_protein_ligand_interactions/talktorial.ipynb

```diff
@@ -4,58 +4,83 @@
    "cell_type": "markdown",
```

Comment:
<p>Maybe we can find a more overview like figure that combines the different interactions? The individual figures are linked in the pdf already anyways (btw I really like the showing of the pdf in the nb!!!)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Oh actually how about we simply use the figure for 3poz (or a protein of your choice) when using the website? Maybe also the tables they show can be reused in the talktorial for explanation?</p>